ktlint-ruleset-experimental/src/test/resources/spec/indent/format-dot-qualified-expression-expected.kt.spec

```diff
@@ -1,10 +1,9 @@
 class A {
-    private fun getImplementationVersion() =
-        javaClass.`package`.implementationVersion
-            ?: javaClass.getResourceAsStream("/META-INF/MANIFEST.MF")
-                ?.let { stream ->
-                    Manifest(stream).mainAttributes.getValue("Implementation-Version")
-                }
+    private fun getImplementationVersion() = javaClass.`package`.implementationVersion
```

Comment:
For functions like this, I actually think I like that Ktlint forces you to put your function body starting on a new line. It reads better.

Reply:
Yeah in this case I'm sort of +/- on the change. Really the case that seems broken is:
```
val foo = someClass
    .aFunction()
    .anotherFunction()
```

becomes
```
val foo = 
    someClass
        .aFunction()
        .anotherFunction()
```

So maybe this is too heavy a hammer.

Reply:
@JLLeitschuh one thing to remember is that with this PR, ktlint won't disallow the code as it was originally formatted, it just won't force it.